### DISCUSSION TITLE
Mujoco 220 and Mujoco-py 210

### DISCUSSION THREAD

**Discussion by Zhaohhya** [google-deepmind/mujoco#1562] [General]
Here I would like to share a problem I encountered, when using gymnasium,
` env = gym.make('Ant-v4', ctrl_cost_weight=0.1)`,
 this line of code will report an error, 
`You appear to be missing MuJoCo. We expected to find the We expected to find the file here: C:\Users\Administrator\.mujoco\mujoco210 `
in fact I'm using mujoco220, I checked and found that I had installed mujoco-py 210 at some point which was causing the error, so I uninstalled mujoco-py, and it ran successfully!

**Comment by rodriguesvinay** [comment]
Thank you for your comment. I was struggling with this error for 3 hours!
